modules/joinmarket-ob-watcher.nix

```diff
@@ -100,7 +95,7 @@ in {
         SystemCallFilter = nbLib.defaultHardening.SystemCallFilter ++ [ "mbind" ] ;
         Restart = "on-failure";
         RestartSec = "10s";
-      } // nbLib.allowTor;
+      } // nbLib.allowAllIPAddresses;
```

Comment:
> because it doesn't make sense to lock down these Tor-only services.

Why not? Locking down is good, even if the service promises to be Tor-only.

Reply:
Fixed.